ups/coadd_utils.cfg

```diff
@@ -3,7 +3,7 @@
 import lsst.sconsUtils
 
 dependencies = {
-    "required": ["afw", "pex_logging"],
+    "required": ["afw"],
     "buildRequired": ["swig"],
```

Comment:
Is it not necessary to replace "pex_logging" with "log" (as you did here https://github.com/lsst/daf_persistence/commit/04c1827ac69db523c251f51d32add72b9d1d606a#diff-8ffc87142b5d2a853222efb750057a1eL6) that no C code uses logging in this repo?